cmd/block/analyze/process.go

```diff
@@ -63,8 +63,12 @@ func (c *command) process(ctx context.Context) error {
 	// Calculate how many parents we need to fetch.
 	minSlot := slot
 	for _, attestation := range attestations {
-		if attestation.Data.Slot < minSlot {
-			minSlot = attestation.Data.Slot
+		attestData, err := attestation.Data()
```

Comment:
Is there a reason to use `attestData` here rather than the expected `attestationData` that you have used elsewhere?

Reply:
Yes there is an `attestationData` struct in this [package](https://github.com/wealdtech/ethdo/blob/master/cmd/block/analyze/command.go#L102-L106) that causes a conflict. 

I can rename the struct to something like `attestationDataInfo` and make the variables consistent if you prefer?

Reply:
Yeah it jarred when reading it, so tweaking the name of the struct and standardising the variable name makes sense here.